### DISCUSSION TITLE
[TOOL] Keira3 released!

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [azerothcore/azerothcore-wotlk#3890] [Tools]
After more than one year of active development and testing, we've finally reached our first stable release of Keira3.

Keira 3.0.0 features a stable, feature-rich editor that has been already used by hundreds of developers, according to the [download statistics](https://somsubhra.com/github-release-stats/?username=azerothcore&repository=Keira3) of the development releases (alpha/beta/RC).

We'd like to thank all contributors that have been using Keira3 so far and reporting issues/suggestions.

### Keira3 Website

For this occasion, we created a new web page containing an overview of Keira3's features, available at:

https://www.azerothcore.org/Keira3

### Support us

- [Donate with PayPal](https://www.paypal.me/francesco92dev)

![image](https://user-images.githubusercontent.com/75517/94994398-a4527500-0597-11eb-820c-7d8ccf728887.png)

**Comment by Viste** [comment]
nice!

**Comment by chaosua** [comment]
Nice job, heard a lot about Keira2 and how hard it was to setup

**Reply by Helias** [reply]
Hi! Keira2 was usable on the web so if you do not install it from scratch was from some point of view very easy to use, unfortunately, it was working with local API + web app which makes it a bit tough to configure, but Keira3 is really download and play like Quice/Truice.
